TextAnalyzer/TextFileFetcher.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Net;
+using System.IO;
+
+namespace TextAnalyzer
+{
+    static class TextFileFetcher
+    {
+        private const string FILE_NAME = "text.txt";
+        private const string LINK = "https://s3.zylowski.net/public/input/3.txt";
+
+        // Fetch the text, download if needed
+        public static string GetTextFileString()
+        {
+            if (File.Exists(FILE_NAME))
+            {
+                return File.ReadAllText(FILE_NAME);
+            }
+
+            using (var client = new WebClient())
+            {
+                try
+                {
+                    client.DownloadFile(LINK, FILE_NAME);
+                }
+                catch(Exception)
+                {
+                    Console.Write("Could not fetch the file, try again.");
+                    return null;
+                }
+            }
+
+            if (File.Exists(FILE_NAME)) // Failsafe condition
+            {
+                return File.ReadAllText(FILE_NAME);
+            }
+            return null;
+        }
+
+        public static void CleanupFileIfPossible()
```

Comment:
/// <summary>
        /// Clean up file if it's possible.
        /// </summary>

Dodaj summary.

Reply:
Uważam że dodawanie komentarza który właściwie tylko powtarza nazwę metody jest zbędne.